### PR TITLE
Added counting of entries in filter type menu items

### PR DESCRIPTION
**Pull request by areksolek**
image

### CODE REVIEW

File: public_html/layouts/basic/modules/Vtiger/resources/Menu.js [hunk 1]
```diff
@@ -28,6 +28,21 @@ jQuery.Class('Vtiger_Menu_Js', {
...
+
+	/**
+	 * Gets the number of entries according to the selected filter
+	 */
+	registerRecordsCount() {
+		$('.js-count').each(function (index, element) {
+			let countEntries = $(element);
+			if (countEntries.length > 0) {
+				AppConnector.request(countEntries.data('url') + '&action=Pagination&mode=getTotalCount').done(function (data) {
+					countEntries.addClass('badge count badge badge-danger c-badge--md ml-2');
```
Comment (rskrzypczak) [inline public_html/layouts/basic/modules/Vtiger/resources/Menu.js]: move and change ml-1

File: public_html/layouts/basic/modules/Vtiger/resources/Menu.js [hunk 2]
```diff
@@ -28,6 +28,21 @@ jQuery.Class('Vtiger_Menu_Js', {
...
+					countEntries.html(JSON.parse(data).result.totalCount);
```
Comment (rskrzypczak) [inline public_html/layouts/basic/modules/Vtiger/resources/Menu.js]:
```suggestion
					countEntries.text(JSON.parse(data).result.totalCount);
```

**Review by rskrzypczak — CHANGES_REQUESTED** [review]

File: public_html/layouts/basic/modules/Vtiger/resources/Menu.js [hunk 3]
```diff
@@ -28,6 +28,21 @@ jQuery.Class('Vtiger_Menu_Js', {
 				}
 			});
 		if (tabsWidth < largeNav) jQuery('#commonMoreMenu').hide();
+	},
+
+	/**
+	 * Gets the number of entries according to the selected filter
+	 */
+	registerRecordsCount() {
+		$('.js-count').each(function (index, element) {
```
Comment (mariuszkrzaczkowski) [inline public_html/layouts/basic/modules/Vtiger/resources/Menu.js]:
```suggestion
		$('.js-menu__content .js-count').each(function (index, element) {
```

File: public_html/layouts/basic/modules/Vtiger/resources/Menu.js (hunk 1 above)
Comment (mariuszkrzaczkowski) [inline public_html/layouts/basic/modules/Vtiger/resources/Menu.js]:
```suggestion
					countEntries.addClass('count badge badge-danger c-badge--md ml-2');
```

**Review by mariuszkrzaczkowski — CHANGES_REQUESTED** [review]

**Pull request merged**